packages/ifpack2/src/Ifpack2_RILUK_decl.hpp

```diff
@@ -640,10 +640,9 @@ clone (const Teuchos::RCP<const NewMatrixType>& A_newnode) const
   RCP<ParameterList> plClone = Teuchos::parameterList ();
```

Comment:
This code occurs in the `clone` method, which we should actually deprecate as well.

Reply:
Deprecating Ifpack2's clone methods (which clone preconditioners) is beyond the scope of this PR.  It is more relevant to #4900.